save-plugins/fix-plugin/src/commonTest/kotlin/org/cqfn/save/plugins/fix/FixPluginTest.kt

```diff
@@ -97,6 +97,52 @@ class FixPluginTest {
         }
     }
 
+    @Test
+    @Suppress("TOO_LONG_FUNCTION")
+    fun `test for batchSize`() {
+        val config = fs.createFile(tmpDir / "save.toml")
+        val testFile1 = fs.createFile(tmpDir / "Test3Test.java")
+        val testFile2 = fs.createFile(tmpDir / "Test4Test.java")
+        fs.write(testFile1) {
+            write("Expected file".encodeToByteArray())
```

Comment:
Did you mean `Original file`?
Why test files and expected files contain the same content before FixPlugin execution?

Reply:
Fixed